pages/common/git-contrib.md

```diff
@@ -0,0 +1,9 @@
+# git contrib
+
+> Display commits from a specific author.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-contrib>.
+
+- Display all commit hashes and corresponding commit messages from a specific author/username:
```

Comment:
```suggestion
- Display all commit hashes and corresponding commit messages from a specific author:
```
`from a specific username` sounds weird and it's already implied that every author has to have a usermane.